drv/gimlet-hf-server/src/main.rs

```diff
@@ -61,6 +61,14 @@ struct Config {
     pub clock: u8,
 }
 
+// There isn't a great crate to do `From` implementation so do this manually
```

Comment:
I have a _slight_ bias towards making `drv-hf-api` depend on `drv-stm32h7-qspi`, so that it can define the `From` implementation, but I don't feel strongly about it.

Reply:
I suppose the downside of that is that it probably makes `drv-hf-api` not compile for target MCUs other than STM32H7. But, we don't currently have any reason to implement this API on other MCUs, so :woman_shrugging:

Reply:
We also use `drv-hf-api` on cosmo which is still an STM32H7 but does not use `drv-stm32h7-qspi` for host flash so it felt a little weird to add that there. I don't love any option here tbh.

Reply:
The Industrial Strength version would be to add a `stm32` feature to `drv-hf-api` which then brings in the QSPI crate and adds the `From` implementation, but that's Even More Shenanigans 🤷🏻

Reply:
alternatively `QspiError` could go in some kinda `qspi-types` crate or something that both the `drv-stm32h7-qspi` and `drv-hf-api` crates depend on. but...that's a Lot...